support/packages/github-to-jira-synchronization-tool/src/config/config.js

```diff
@@ -3,21 +3,45 @@
  * SPDX-License-Identifier: MIT
  */
 
+const fs = require('fs');
+const path = require('path');
+
 let config = null;
+let lastModifiedTime = 0;
+
+const CONFIGURATION_PATH = path.resolve(__dirname, '../../mapping-config.json');
+
+function fileHasChanged() {
```

Comment:
how can this change? do you modify it manually in the deployment environment without redeploying? :thinking:

Reply:
Yep, that was my idea. This can happen if Bruno has to add some user mapping while the app is running. This way we can avoid redeploying the app for that minor change